app/src/main/java/com/g11x/checklistapp/data/Database.java

```diff
@@ -58,6 +65,13 @@ public Uri insert(ContentValues contentValues) {
         long id = db.insert(Database.ImportantInformation.TABLE_NAME, null, contentValues);
         return ContentUris.withAppendedId(contentUri, id);
       }
+
+      @Override
+      public Cursor query(String[] projection, String selection, String[] selectionArgs,
```

Comment:
Yea the only thing that comes to mind is default projections or something. Definitely a good thing to keep in mind.